rest-api/offline/biobank_samples_pipeline.py

```diff
@@ -235,24 +239,20 @@ def _query_and_write_reports(exporter, now, path_received, path_late, path_missi
        database_factory.get_database().session() as session:
     writer = CompositeSqlExportWriter([received_writer, late_writer, missing_writer])
     exporter.run_export_with_session(writer, session, replace_isodate(_RECONCILIATION_REPORT_SQL),
-                                     {"biobank_id_prefix": get_biobank_id_prefix(),
-                                      "pmi_ops_system": _PMI_OPS_SYSTEM,
-                                      "kit_id_system": _KIT_ID_SYSTEM,
-                                      "tracking_number_system": _TRACKING_NUMBER_SYSTEM})
+                                     {'race_question_code_id': race_question_code.codeId,
+                                      'native_american_race_code_id':
```

Comment:
opt_nit: Doesn't this include "alaskan natives" as well (which I gather is distinct from native american)? Should this keep the `aian` naming here and throughout?

Reply:
Added comment to [DA-354] inquiring about this.